gh-repo-stats

```diff
@@ -213,68 +215,25 @@ Header() {
   echo "######################################################"
   echo ""
 
-  ############################################
-  # Clean any whitespace that may be entered #
-  ############################################
-  GHE_URL_NO_WHITESPACE="$(echo -e "${GHE_URL}" | tr -d '[:space:]')"
-  GHE_URL=$GHE_URL_NO_WHITESPACE
-
-  #################################
-  # Get the Personal Access Token #
-  #################################
-  GetPersonalAccessToken
-
-  ################
-  # Set the URLS #
-  ################
-  if [[ "${GHE_URL}" == "https://github.com" ]]; then
-    GITHUB_URL="https://api.github.com"
-    GRAPHQL_URL="https://api.github.com/graphql"
-  else
-    GITHUB_URL="${GHE_URL}/api/v3"
-    GRAPHQL_URL="${GHE_URL}/api/graphql"
-  fi
-
-  if [[ -z "${REPO_PAGE_SIZE}" ]]; then
-    REPO_PAGE_SIZE=20
-  fi
-
-  if [[ -z "${EXTRA_PAGE_SIZE}" ]]; then
-    EXTRA_PAGE_SIZE=100
-  fi
-
   ################################################################
   # Validate we can hit the endpoint by getting the current user #
   ################################################################
   if [[ ${GITHUB_TOKEN_TYPE} == "user" ]]; then
-    Debug "curl -kw '%{http_code}' -s -X GET \
-    --url ${GITHUB_URL}/user \
-    -H \"authorization: Bearer ************\""
-  
-    USER_RESPONSE=$(curl -kw '%{http_code}' -s -X GET \
-      --url "${GITHUB_URL}"/user \
-      -H "authorization: Bearer ${GITHUB_TOKEN}")
-  
-    Debug "USER_RESPONSE: "
-    Debug "${USER_RESPONSE}"
-  
-    USER_RESPONSE_CODE="${USER_RESPONSE:(-3)}"
-    USER_DATA="${USER_RESPONSE::${#USER_RESPONSE}-4}"
-  
-    DebugJQ "${USER_DATA}"
+    USER_DATA=$(gh api /user)
+    ERROR_CODE=$?
```

Comment:
Perhaps a better name for this variable is `EXIT_CODE` since it's only an error code if non-zero.